### PR TITLE
docs(release): update list of tools and maintainers to notify when releasing

### PR DESCRIPTION
**Pull request by smoya**
This PR updates `RELEASE_PROCESS.md` file. In particular, the list of repositories whose owners should be notified when prereleasing.

### CODE REVIEW

**Review by fmvilas — APPROVED** [review]
I haven't checked if that's the whole list of tools but the one you're proposing looks good to me 👍

**Review by dalelane — APPROVED** [review]

**Comment by smoya** [comment]
@derberg whenever you have a chance, could you review this? No rush 👍

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -309,23 +318,44 @@ First, changes are merged into the `spec` repository, then `spec-json-schemas` a
...
 
 
-### Step 16 - notify tool maintainers
-
-The [release coordinator](#%22release-coordinator%22) should notify maintainers of the following repositories that the first feature is merged and that release will be produced and therefore they need to start preparing for it:
+### Step 17 - notify tool maintainers
+
+Our current CI/CD automation will fill PR's updating the dependencies **automatically** on all repositories after the release. 
+However, the [release coordinator](#%22release-coordinator%22) should notify maintainers of the dependant repositories that a new release happened, as those might want to adopt the new features. 
+Here is the [full list of repositories depending on parser-js or specs](https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Aasyncapi+%28path%3Apackage.json+OR+go.mod%29+%22%40asyncapi%2Fparser%22+OR+%22%40asyncapi%2Fspecs%22+OR+%22github.com%2Fasyncapi%2Fspec-json-schemas%22+OR+%22github.com%2Fasyncapi%2Fparser-go%22#). Some of them are:
```
Comment (derberg) [inline RELEASE_PROCESS.md:330]: https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Aasyncapi+%28path%3Apackage.json+OR+go.mod%29+%22%40asyncapi%2Fparser%22+OR+%22%40asyncapi%2Fspecs%22+OR+%22github.com%2Fasyncapi%2Fspec-json-schemas%22+OR+%22github.com%2Fasyncapi%2Fparser-go%22# is not a publicly available resource, at least not for me, so also by default not for others

Comment (smoya) [reply RELEASE_PROCESS.md:330]: I don't have an alternative for this. I couldn't do the same search in the standard GH. Code Search is not public yet for the whole public.

**Review by derberg — APPROVED** [review]

**Comment by smoya** [comment]
@dalelane @fmvilas since your last review, I provided an alternative to GH Code Search to query for code dependants on the spec and parsers. https://github.com/asyncapi/spec/pull/770/commits/d7dc58f41efe0721150d35f47fb428492dd2fb27

Would you mind checking and, if all good, approving again? Thanks! 🙏

**Review by derberg — APPROVED** [review]

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -309,23 +318,52 @@ First, changes are merged into the `spec` repository, then `spec-json-schemas` a
...
+  - [NodeJS WS Template](https://github.com/asyncapi/nodejs-ws-template)
+  - [Optimizer](https://github.com/asyncapi/optimizer)
+  - [Parser Go](https://github.com/asyncapi/parser-go)
+  - [React component](https://github.com/asyncapi/asyncapi-react/)
+  - [Server API](https://github.com/asyncapi/server-api/)
+  - [Simulator](https://github.com/asyncapi/simulator)
+  - [Studio](https://github.com/asyncapi/studio)
+  - [TS NATS Template](https://github.com/asyncapi/ts-nats-template)
+
+You can use Github Code Search to find the [list of repositories depending on parser-js or the specs](https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Aasyncapi+%28path%3Apackage.json+OR+go.mod%29+%22%40asyncapi%2Fparser%22+OR+%22%40asyncapi%2Fspecs%22+OR+%22github.com%2Fasyncapi%2Fspec-json-schemas%22+OR+%22github.com%2Fasyncapi%2Fparser-go%22#).
```
Comment (dalelane) [inline RELEASE_PROCESS.md:354]: I'm still on the waiting list for code search so I can't try this, but it's a neat idea

**Review by dalelane — APPROVED** [review]
nice - that's a neat approach 👍

**Review by derberg — APPROVED** [review]

**Review by dalelane — APPROVED** [review]

**Comment by fmvilas** [comment]
Forget about the last comment saying it was released in version 2.5.0-next-major-spec.1. I made a mistake and it created this version but it should actually be `3.0.0-next-major-spec.1`. There's a notice in the [release](https://github.com/asyncapi/spec/releases/tag/v2.5.0-next-major-spec.1). Apologies for the noise.

**Comment by derberg** [comment]
Recent comments about the release from the bot were added by mistake. More details in https://github.com/asyncapi/spec/issues/899

**Pull request merged**